app/src/main/java/com/yagni/view/ManageInvGUI.java

```diff
@@ -9,365 +9,447 @@
             updateRecord.update(linkDB);
+
+            // Reload product inventory data table to display updated data
             refreshProducts();
 
-		} else if (e.getSource() == reloadButton) {
+        } else if (e.getSource() == reloadButton) {
+            // On click of the reload button refresh the products table
             refreshProducts();
 
         } else if (e.getSource() == logoutButton) {
-            tablePanel.remove(reloadButton);
-            tablePanel.repaint();
-            tablePanel.add(loadButton);
+            // On click of the logout button close the db connection and dispose the
+            // JFrame
             linkDB.closeConnection();
-
-            clearProductsTable();
             inventoryFrame.dispose();
-            
+
+            // Create a new HomeGUI object to display the Home page
             new HomeGUI();
 
         } else if (e.getSource() == backButton) {
-            tablePanel.remove(reloadButton);
-            tablePanel.repaint();
-            tablePanel.add(loadButton);
-            
-            clearProductsTable();
+            // On click of the back button dispose the current JFrame
             inventoryFrame.dispose();
-            
+
+            // Passing the db connection, with new object to render Employee GUI page
             new EmployeeGUI(linkDB);
 
         } else if (e.getSource() == searchButton) {
+            // On click of the search button clear all displayed products in the JTable
             clearProductsTable();
-            String productId = searchField.getText();
-            Search searchObj = new Search(productId, productsTable); // creates a searchObj object from model/Search.java and passes in the products table
-            searchObj.readOne(linkDB); // calls the readOne method and passes in the database connection
+
+            // set productId to the user input
+            final String productId = searchField.getText();
+
+            // Create a search object passing in productID and products table
+            Search searchObj = new Search(productId, productsTable);
+
+            // call the readOne method and pass through the db connection
+            searchObj.readOne(linkDB);
 
         } else if (e.getSource() == deleteButton) {
-            
-            String deleteID = productIDField.getText();
-            Delete deleteObj = new Delete(deleteID); //object of delete to get the id we want to delete.
-            deleteObj.delete(linkDB);
-            int action = JOptionPane.showConfirmDialog(null, "Do you really want to delete this product?\nThis action cannot be undone.","Delete", JOptionPane.YES_NO_OPTION, 3, warningImg);
+            // On click of delete, set variable to user input from productID text field
+            final String deleteID = productIDField.getText();
+
+            // JOptionPane to display a pop up to the user to confirm product delete
+            final int action = JOptionPane.showConfirmDialog(null,
```

Comment:
I really like the line wrapping on Line 338-340. You have proper indentation here too. I'll make sure to mention this for other's to reference when dealing with our long SQL statements.

Reply:
Thanks! The auto formatter has been really helpful with that!!